api/src/main/resources/org/openmrs/liquibase/updates/liquibase-update-to-latest-2.6.x.xml

```diff
@@ -133,5 +133,20 @@
 		<comment>Adding not null constraint to encounter_diagnosis.dx_rank. TRUNK-6081</comment>
 		<addNotNullConstraint tableName="encounter_diagnosis" columnName="dx_rank" columnDataType="int"/>
 	</changeSet>
+
+	<changeSet id="TRUNK-6204-2023-10-23" author="manojll">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="orders"/>
```

Comment:
Shouldn't these changes be in `liquibase-update-to-latest-2.7.x.xml`?

Reply:
@dkayiwa There is no file like `liquibase-update-to-latest-2.7.x.xml`.

Reply:
That is why i created this ticket: https://issues.openmrs.org/browse/TRUNK-6207

Reply:
Now that the above ticket is merged, can you move the changeset to the appropriate file?

Reply:
I will open new Pr for this ticket